optuna/storages/grpc/_auto_generated/api_pb2_grpc.py

```diff
@@ -0,0 +1,916 @@
+# mypy: ignore-errors
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
+"""Client and server classes corresponding to protobuf-defined services."""
+import grpc
+import warnings
+
+import optuna.storages.grpc._auto_generated.api_pb2 as api__pb2
```

Comment:
I fixed this line. The originally generated line is
```
import api_pb2 as api__pb2
```
This does not work. According to https://github.com/grpc/grpc/issues/29459, if we create a directory for the proto files in the top of the optuna project like `proto/optuna/storages/grpc/_auto_generated/api.proto`, we can avoid to fix this line, but I don't think it's reasonable way. Do you have any opinions?

Reply:
Thank you for your explanation! Personally, I prefer placing the proto file in `optuna/storages/grpc/_proto/api.proto` and generating the Python files there to avoid manual fixes.

That said, this approach is reasonable and acceptable, as the modification is simple enough to handle with a sed command. I'll proceed with the review.

Reply:
@mist714 Do you have an opinion on this?

Reply:
I think it's fine as a temporary measure. 
I think it's correct to control it depending on directory where `protoc` is executed, so how about leaving this as a TODO comment?